src/effects/nyquist/Nyquist.cpp

```diff
@@ -1387,12 +1388,12 @@ bool NyquistEffect::ProcessOne()
 
    // Put the fetch buffers in a clean initial state
    for (size_t i = 0; i < mCurNumChannels; i++)
-      mCurBuffer[i].Free();
+      mCurBuffer[i].reset();
 
```

Comment:
Was using Free here previously a bug?  A comment about why using reset() is worth having.

Reply:
See the header file.  I change the type of mCurBuffer.  reset() is the equivalent in the new type for what Free() formerly did.